FactoryWeather/App/VIP Home/HomeViewController.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  HomeViewController.swift
+//  FactoryWeather
+//
+//  Created by Matej Korman on 08/01/2019.
+//  Copyright © 2019 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+protocol HomeDisplayLogic: class {
+    func displayWeather(_ weather: HomeContentView.ViewModel, forLocation location: Location)
+}
+
+protocol HomeSceneLogic: class {
+    func getWeather(forLocation location: Location)
+    func setSearchTextFieldHidden(_ hidden: Bool)
+}
```

Comment:
I quite like how we have the two protocols, each defining it's own thing - the `View` capabilities of the `View` layer and the high-level scene capabilities of the scene 👍 👍